web-app/README.md

```diff
@@ -0,0 +1,27 @@
+# Faucet Web App
+
+This is a next.js web app that is used to interact with the faucet smart contract.
+We ask the user to connect using their wallet and a OAuth provider (GitHub or Discord) to protect against spam requests.
+
+## Environment Variables
+
+| Variables | Description |
+| -------- | ------- |
+| `NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID` | ProjectID use by WalletConnect connector https://cloud.walletconnect.com/ |
+| `GITHUB_CLIENT_ID` | GitHub App id https://github.com/settings/apps |
+| `GITHUB_CLIENT_SECRET` | GitHub App Secret |
+| `GITHUB_ACCOUNT_NAME` | Account name on GitHub we will be looking at to make sure the user is following this account before requesting token |
```

Comment:
In this case we will be setting to the Subspace org?

Reply:
Exactly